lib/slayer/form.rb

```diff
@@ -1,17 +1,31 @@
+require 'virtus'
+require 'dry-validation'
+
 module Slayer
   class Form
-    extend ModelAttribute
-    include ActiveModel::Validations if defined?(Rails)
+    attr_reader :errors
+    include Virtus.model
 
-    # Internal: Form Class Methods
     class << self
-      def from_params(params)
-        raise NotImplementedError
-      end
+      attr_accessor :_schema
 
-      def from_model(model)
-        raise NotImplementedError
+      def validations(&block)
```

Comment:
It would be nice to have a system where you can also set the Schema directly. In the event that you have a couple of Forms that share a schema (but are slightly different), you could define the schema externally and directly re-use it on both objects (without having to repeat the block, or store the block as a proc somewhere).